scripts/build.js

```diff
@@ -1,5 +1,7 @@
+#!/bin/env node
+/* eslint-disable no-console */
```

Comment:
NIT: could just use `eslint-disable-next-line` for the one existing use case, so this file won't get muddied up by accident :)

...but I'm not losing any sleep over it.

Reply:
I was thinking about that but felt a cli script should always allow `console.log` anyway.